update_table.py

```diff
@@ -0,0 +1,133 @@
+        if balances.get(symbol, 0) > 0:
+            update.append([
+                date,  # Update date
+                balances_names[idx],  # Asset name
+                symbol,  # Asset symbol
+                prices.get(symbol).get('USD'),  # Asset USD price
+                prices.get(symbol).get('BTC'),  # Asset BTC price
+                float(prices.get(symbol).get('BTC')) / float(prices.get('ETH').get('BTC')),  # Asset ETH price
+                balances.get(symbol),  # Asset balance
+                portfolio.get(symbol).get('USD'),  # Asset USD value
+                portfolio.get(symbol).get('BTC'),  # Asset BTC value
+                portfolio.get(symbol).get('ETH'),  # Asset ETH value
+                portfolio.get(symbol).get('share'),  # Asset % share
+            ])
+    api.add_portfolio(update)
+    return portfolio
+
+
+def update_daily_performance(api, date, portfolio):
+    # Updates current portfolio
+
+    initial_token_usd_price = 10.0
+    initial_token_btc_price = 0.010103
+    initial_token_eth_price = 0.206957
+
+    portfolio_value = {
+        'USD': sum([value.get('USD') for value in portfolio.values()]),
+        'BTC': sum([value.get('BTC') for value in portfolio.values()]),
+        'ETH': sum([value.get('ETH') for value in portfolio.values()]),
+    }
+
+    latest_token_prices = api.get_latest_token_prices()
+    latest_token_supply = api.get_latest_token_supply()
+
+    number_of_tokens = fetch_token_supply()
+
+    row = [
+        date,  # Date
+        number_of_tokens,  # Token supply
+        number_of_tokens - latest_token_supply,  # 12h change in token supply
+        portfolio_value.get('USD'),  # Portfolio USD price
+        portfolio_value.get('BTC'),  # Portfolio BTC price
+        portfolio_value.get('ETH'),  # Portfolio ETH price
+        portfolio_value.get('USD') / number_of_tokens,  # USD Token price
+        portfolio_value.get('USD') / number_of_tokens / latest_token_prices[0] - 1,  # USD price 12h change
+        portfolio_value.get('USD') / number_of_tokens / initial_token_usd_price - 1,  # USD price lifetime change
+        portfolio_value.get('BTC') / number_of_tokens,  # USD Token price
+        portfolio_value.get('BTC') / number_of_tokens / latest_token_prices[1] - 1,  # BTC price 12h change
+        portfolio_value.get('BTC') / number_of_tokens / initial_token_btc_price - 1,  # BTC price lifetime change
+        portfolio_value.get('ETH') / number_of_tokens,  # USD Token price
+        portfolio_value.get('ETH') / number_of_tokens / latest_token_prices[2] - 1,  # ETH price 12h change
+        portfolio_value.get('ETH') / number_of_tokens / initial_token_eth_price - 1,  # ETH price lifetime change
+    ]
+    api.add_daily_performance_row(row)
+
+    return row
+
+
+def update_table(config):
+    api = SheetsAPI(sheets_id=config['sheets']['id'], secret_file=config['sheets']['secret_file'])
+    date = time.strftime('%Y-%m-%d %H:%M %Z')
```

Comment:
I think would be better to convert datetime to UTC and print it without timezone